### PR TITLE
Skip Buttons

### PR DESCRIPTION
**Pull request by nibhar**
Skip buttons now only appear on:

- change-password's file download page to skip the download of the file
- login's file download page after word import to skip the download of the file
- complete export's file-completed page to skip recovery words

Styles for the skip button are moved to common.css.

### CODE REVIEW

File: src/request/export/ExportFile.js [hunk 1]
```diff
@@ -66,11 +66,7 @@ class ExportFile extends Nimiq.Observable {
                 hideCancel: true,
             },
         );
-        this._passwordSetterBox = new PasswordSetterBox(
-            $passwordSetterBox, {
-                hideSkip: true,
-            },
-        );
+        this._passwordSetterBox = new PasswordSetterBox($passwordSetterBox);
```
Comment (sisou) [inline src/request/export/ExportFile.js]: There should be no need for a password setter during export anymore, when keys cannot be unencrypted anyway. But we might want to keep it in anyway, to be sure?

File: src/request/export/ExportWords.js [hunk 2]
```diff
@@ -72,21 +68,12 @@ class ExportWords extends Nimiq.Observable {
         /* eslint-enable no-new */
 
         // events
-        $noRecoverySkip.addEventListener('click', event => {
-            event.preventDefault();
-            this._resolve({ success: false });
-        });
-        $recoveryWordsSkip.addEventListener('click', event => {
```
Comment (sisou) [inline src/request/export/ExportWords.js]: Do we really want to remove the skip for the word validation? I liked that one.

Comment (sisou) [reply src/request/export/ExportWords.js]: As discussed, to not confuse users, word validation will be non-skippable.

File: src/request/export/ExportFile.js (hunk 1 above)
Comment (nibhar) [reply src/request/export/ExportFile.js]: As discussed previously, I did not remove the functionality. It is just no longer reachable.

**Pull request merged**